### ISSUE TITLE
Why did you remove the code?

### ISSUE THREAD

**Issue by jamezrin** [ndg63276/alexa-youtube#143]
Why out of the sudden you decided to privatize the code? It was fine like it was...

**Comment by ndg63276** [comment]
Unfortunately more people were stealing my code and trying to profit off of it, than were trying to collaborate to help improve it. So I've removed the code. Sorry about that. Yeah I should probably delete the history, but I deleted the files in a hurry over Christmas.

**Comment by ndg63276** [comment]
Ok, I think I've removed the important files from the history.

**Comment by HeIp** [comment]
I don't find a good point in this decision.
For the following reasons:
1)There's copies of this all over the internet now(before code removal)
2)There's Forks of earlier versions of the code
3)People Can just got here : https://github.com/ndg63276/alexa-youtube/tree/a4e848c7dfdbda834393e8f3b7770cf4ceb354dd

**Comment by ndg63276** [comment]
No, well, people with forks are welcome to continue development of the code themselves. Generally though, people weren't very helpful. I've had 16 pull requests in 2 years, and most were just language updates. And then other people were profiting off my work. So I've decided it should only be me profiting off it. You can call me selfish if you want.

As for point 3, I've just removed the lambda_function.(py|zip) files from the commits, so it's not much use to anyone.

**Comment by jamezrin** [comment]
I don't think you are selfish by doing this, but I don't think it makes sense. People will still be profiting off your work nonetheless, just not from your future work as you privatized it. 

Who has been profiting off your work? I would like to know. If it's public, maybe try DMCAing it, in theory as you didn't include any license notice in your code, it is copyrighted to you.

**Comment by marmocchio8** [comment]
Muori Perfavore

**Comment by mariodmmariodm** [comment]
ah, I understand this decision but I'm very sorry. In my opinion, Amazon should have allowed you to publish them officially as done in other countries. How is it going, does it continue to work privately or is it blocked? my free aws account expired and therefore I had stopped using it, but it was very useful as a skill

**Comment by crd** [comment]
I was disappointed to notice that you had taken the project private but I understand the motivation -- I'm sorry to hear that people were profiting off of your work. Thanks for your work, sorry to see this shut down. Tinkering with it was a good learning experience for me.

**Comment by nguyen-k** [comment]
Yikes.  I'm glad I found this discussion; spent a good hour trying to find that dang 'ol lambda file. So disappointing as a DIY person.

**Comment by timguy** [comment]
@nguyen-k where did you found it? Or did you or anybody else found a similar project?

**Comment by lkraider** [comment]
https://github.com/crd/jukebox
or
https://github.com/dmhacker/alexa-youtube-skill (discontinued but has good docs)

**Comment by haithamalbadi** [comment]
We need mew update please 😭😭😭
